gwcs/coordinate_frames.py

```diff
@@ -382,51 +382,53 @@ class TemporalFrame(CoordinateFrame):
 
     Parameters
     ----------
-    axes_order : tuple or int
-        A dimension in the input data that corresponds to this axis.
-    reference_frame : `object`
-        The object to instantiate to represent the time coordinate. Defaults to
-        `astropy.time.Time`. Use partial functions to customise the
-        `~astropy.time.Time` instance.
-    reference_time : `astropy.time.Time` or `None`
-        Reference time, the time of the 0th coordinate. If none the values of
-        the axis are assumed to be valid times.
-    unit : str or units.Unit instance
-        Spectral unit.
+    reference_frame : `~astropy.time.Time`
+        A Time object which holds the time scale and format.
+        If data is provided, it is the time zero point.
+        For relative times `data` should be an empty list.
```

Comment:
I initially read this as the model returning relative times rather than the frame. Maybe reword it so it's something like "to not set a zero point for the frame"?